lib/osml/tile_server/ts_dataplane.ts

```diff
@@ -241,22 +250,6 @@ export class TSDataplane extends Construct {
       }
     });
 
-    // Create a Lambda function to clean up the TS DLQ
```

Comment:
Wrong comment. But the below lambda function is not needed since it has been moved to TSTestRunner class